airbyte-integrations/connectors/source-sendgrid/source_sendgrid/manifest.yaml

```diff
@@ -1,257 +1,980 @@
-version: "0.29.0"
-definitions:
-  page_size: 50
-  step: "P30D"
-
-  schema_loader:
-    type: JsonFileSchemaLoader
-    file_path: "./source_sendgrid/schemas/{{ parameters.name }}.json"
-
-  requester:
-    type: HttpRequester
-    url_base: "https://api.sendgrid.com"
-    http_method: "GET"
-    authenticator:
-      type: "BearerAuthenticator"
-      api_token: "{{ config.apikey }}"
-  cursor_paginator:
-    type: DefaultPaginator
-    page_size: "#/definitions/page_size"
-    limit_option:
-      inject_into: "request_parameter"
-      field_name: "page_size"
-    page_token_option:
-      type: RequestPath
-    pagination_strategy:
-      type: "CursorPagination"
-      cursor_value: "{{ response._metadata.next }}"
-  offset_paginator:
-    type: DefaultPaginator
-    $parameters:
-      page_size: "#/definitions/page_size"
-    limit_option:
-      inject_into: "request_parameter"
-      field_name: "limit"
-    page_token_option:
-      type: RequestOption
-      inject_into: "request_parameter"
-      field_name: "offset"
-    pagination_strategy:
-      type: "OffsetIncrement"
-  retriever:
-    type: SimpleRetriever
-  incremental_sync:
-    type: "DatetimeBasedCursor"
-    start_datetime:
-      datetime: "{{ config['start_time'] }}"
-      datetime_format: "%s"
-    end_datetime:
-      datetime: "{{ now_utc() }}"
-      datetime_format: "%Y-%m-%d %H:%M:%S.%f%z"
-    step: "#/definitions/step"
-    start_time_option:
-      field_name: "start_time"
-      inject_into: "request_parameter"
-    end_time_option:
-      field_name: "end_time"
-      inject_into: "request_parameter"
-    datetime_format: "%s"
-    cursor_granularity: "PT1S"
-  messages_incremental_sync:
-    type: "DatetimeBasedCursor"
-    start_datetime:
-      datetime: "{{ config['start_time'] }}"
-      datetime_format: "%s"
-    end_datetime:
-      datetime: "{{ now_utc() }}"
-      datetime_format: "%Y-%m-%d %H:%M:%S.%f%z"
-    step: "#/definitions/step"
-    datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
-    cursor_granularity: "PT0.000001S"
-
-  base_stream:
-    type: DeclarativeStream
+version: 0.65.0
```

Comment:
This is the version of the connector, not the CDK, right? It just happens to be the same? Should we call the connector version `0.3` or ambitious `1.0` instead?

Reply:
This is what the builder emits at the moment. It aligns with the CDK.

Reply:
It corresponds to the CDK version used by the platform/builder, and is used to determine feature compatibility (i.e. the CDK yells if the manifest was built by a later version than it, since it may use unknown/unsupported features)

Reply:
Upgraded to newest!